dev_requirements.txt

```
@@ -1,6 +1,7 @@
 cython==0.29.21
 setuptools>=27.1.2
 wheel>=0.32.0
+pytest==6.2.4; python_version >= '3.10'
 pytest==5.4.1; python_version >= '3.6'
```

nit: would it be more clear with the following format?
```suggestion
pytest==6.2.4; python_version >= '3.10'
pytest==5.4.1; python_version >= '3.6', < '3.10'
```